rawfile/util.py

```diff
@@ -36,10 +36,12 @@ def wrap(self, request, context):
 
 
 def run(cmd):
+    print(f"{cmd}")
```

Comment:
Please remove this, or make to have a better format

Reply:
Can you suggest the format, currently without these logs it's very difficult to debug
Also there's no timestamps in the current logs, though this print will not address that :)

Reply:
I'm not sure how these things are done in python, but there must be some sort of logging library which is tweakeable to some extent and which would include verbosity levels and timestamps?
I'll remove these for now, but we should think about adding something like that

Reply:
Yeah after filesystems I will change the logging system...